From: James McCoy <jamessan@debian.org>
Date: Sun, 17 Dec 2017 22:15:58 -0500
Subject: syntax/debcontrol.vim: Fix highlighting of javascript Section

When converting to the use of lists instead of long regexps, the
handling of java/javascript was accidentally dropped.

Signed-off-by: James McCoy <jamessan@debian.org>
---
 runtime/syntax/debcontrol.vim | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/runtime/syntax/debcontrol.vim b/runtime/syntax/debcontrol.vim
index b879074..0f5d8d1 100644
--- a/runtime/syntax/debcontrol.vim
+++ b/runtime/syntax/debcontrol.vim
@@ -3,7 +3,7 @@
 " Maintainer:  Debian Vim Maintainers <pkg-vim-maintainers@lists.alioth.debian.org>
 " Former Maintainers: Gerfried Fuchs <alfie@ist.org>
 "                     Wichert Akkerman <wakkerma@debian.org>
-" Last Change: 2017 Nov 04
+" Last Change: 2017 Dec 17
 " URL: https://anonscm.debian.org/cgit/pkg-vim/vim.git/plain/runtime/syntax/debcontrol.vim
 
 " Standard syntax initialization
@@ -52,7 +52,7 @@ let s:sections = [
       \, 'devel', 'doc', 'editors', 'education', 'electronics', 'embedded'
       \, 'fonts', 'games', 'gnome', 'gnustep', 'gnu-r', 'golang', 'graphics'
       \, 'hamradio', 'haskell', 'httpd', 'interpreters', 'introspection'
-      \, 'java', 'javascript', 'kde', 'kernel', 'libs', 'libdevel', 'lisp'
+      \, 'java\%(script\)\=', 'kde', 'kernel', 'libs', 'libdevel', 'lisp'
       \, 'localization', 'mail', 'math', 'metapackages', 'misc', 'net'
       \, 'news', 'ocaml', 'oldlibs', 'otherosfs', 'perl', 'php', 'python'
       \, 'ruby', 'rust', 'science', 'shells', 'sound', 'text', 'tex'
